src/extension/options.js

```diff
@@ -36,6 +36,11 @@ function isValidGitHubURL(giturl) {
     && Object.keys(getGitHubSettings(giturl)).length === 3;
 }
 
+function isValidProdHost(prodHost) {
```

Comment:
I think this should be changed to `isValidHost(host)` now that we are using it to validate other hosts as well.